airbyte-commons-worker/src/main/java/io/airbyte/workers/general/DefaultReplicationWorker.java

```diff
@@ -130,6 +130,12 @@ public final ReplicationOutput run(final StandardSyncInput syncInput, final Path
     destinationConfig.setCatalog(mapper.mapCatalog(destinationConfig.getCatalog()));
 
     final long startTime = System.currentTimeMillis();
+    final long replicationStartTime = startTime;
+    long replicationEndTime;
+    long sourceReadStartTime = -1;
```

Comment:
Why are some of these initialized and some not?

Reply:
Because I'm bad at Java :(
Updated via 3103f2797d51a1aac6b9894444300cac8b893ea3